jax/ops/__init__.py

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+__all__ = [
```

Comment:
What is the value in setting `__all__` if we have already carefully crafted the namespace to contain nothing but exported symbols? It seems like it is just adding another place to update for no value.

Reply:
This is the solution [recommended by PEP8](https://www.python.org/dev/peps/pep-0008/#id50), though I only tend to use it in cases where there aren't too many imports. If you feel strongly about this, I'm happy to change to a ``noqa: F401`` comment instead.

Reply:
I would prefer to leave it out; it just adds busy-work for whoever adds a function. We already created a facade module to solve the visibility problem.

Reply:
OK, Done.